include/arch/arm/aarch32/cortex_m/mpu/arm_mpu_v7m.h

```diff
@@ -262,6 +262,10 @@ typedef struct {
 	{(_K_MEM_PARTITION_P_RX_U_RX   \
 	| NORMAL_OUTER_INNER_NON_CACHEABLE_SHAREABLE)})
 
+#define K_MEM_PARTITION_P_RW_U_NA_DEVICE_SHAREABLE ((k_mem_partition_attr_t) \
```

Comment:
This is fine. I wonder if you can add the declaration for the ARMv8-M architecture as well. 
Optional.

Reply:
No longer needed, therefore removed.